docs/dg/dev/set-up-spryker-locally/key-value-store-configuration.md

```diff
@@ -194,7 +194,7 @@ class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
 To resave storage data, run the command:
 
 ```bash
-SPRYKER_REDIS_IS_DEV_MODE=0 console storage:redis:re-save
+SPRYKER_REDIS_IS_DEV_MODE=0 NEW_RELIC_ENABLED=false console storage:redis:re-save
```

Comment:
we can have the same thing for otel. I'm not sure about this, but it could be

Reply:
you can also mention here: If you have logger or something that can collect data during execution, disable it for this command.